DSCResources/MSFT_xSqlServerInstall/MSFT_xSqlServerInstall.schema.mof

```diff
@@ -19,5 +21,10 @@ class MSFT_xSqlServerInstall : OMI_BaseResource
     [write, Description("Specify the startup account for the SQL server service.")] string SvcAccount;
     [write, Description("Specify logins to be members of the sysadmin role.")] string SysAdminAccounts;
     [write, Description("Specify the account for SQL server agent service.")] string AgentSvcAccount;
+    [write, Description("Specify the default collation for SQL server")] string SqlCollation;
```

Comment:
I thought about adding a Value Map but do you think it is wise given the number of collations available in SQL (across languages)?